### PR TITLE
PPP-Jacob

### PR DESCRIPTION
**Pull request by Jacob-Pang**

### CODE REVIEW

**Review by onnwards — APPROVED** [review]

File: docs/team/jacob-pang.md [hunk 1]
```diff
@@ -9,11 +9,37 @@ App-Ointment is a patient appointment management software adapted from AddressBo
...
+  * Made `Person` an abstract class in order to implemented `Patient` and `Doctor` derived classes.
+  * Added generics into person-dependent storage and processes to accept extensions of the `Person` class, reducing potential duplication of code.
+  * Eased the implementation of `Doctor` class in `v1.3`.
+
+* **New Feature**: Added command for adding appointments.
+  * Enables the user to create an appointment based on existing patient and doctor records.
+  * Designed to allow the user to specify either the duration or the end time of the appointment for flexibility.
 
 * **Project management**:
+  * Reviewed team PRs and provided constructive feedback.
```
Comment (spzj) [inline docs/team/jacob-pang.md]: This should be under Community and include links to the relevant PRs. See the [guide](https://se-education.org/addressbook-level3/team/johndoe.html)

File: docs/team/jacob-pang.md [hunk 2]
```diff
@@ -9,11 +9,37 @@ App-Ointment is a patient appointment management software adapted from AddressBo
...
+  * Implemented appointment-variant storage classes.
+  * Modified the application model and logic to execute appointment-related commands and tasks.
+  * Required an in-depth analysis of the application workflow and logic.
+
+* **Architecture**: Added polymorphism into person-features
+  * Made `Person` an abstract class in order to implemented `Patient` and `Doctor` derived classes.
+  * Added generics into person-dependent storage and processes to accept extensions of the `Person` class, reducing potential duplication of code.
+  * Eased the implementation of `Doctor` class in `v1.3`.
+
+* **New Feature**: Added command for adding appointments.
```
Comment (spzj) [inline docs/team/jacob-pang.md:28]: We need to provide justification/ highlights (if applicable)

File: docs/team/jacob-pang.md [hunk 3]
```diff
@@ -9,11 +9,37 @@ App-Ointment is a patient appointment management software adapted from AddressBo
...
+  * Enables the user to create an appointment based on existing patient and doctor records.
+  * Designed to allow the user to specify either the duration or the end time of the appointment for flexibility.
 
 * **Project management**:
+  * Reviewed team PRs and provided constructive feedback.
+  * Helped to fix critical features and bugs in other portions.
+    * Fixed errors in the initial implementation of the `Doctor` class.
+    * Fixed design flaws in `EditAppointmentCommand`.
 
 * **Documentation**:
```
Comment (spzj) [inline docs/team/jacob-pang.md:37]: Remember to include links to the relevant PRs

File: docs/team/jacob-pang.md (hunk 2 above)
Comment (Jacob-Pang) [reply docs/team/jacob-pang.md:28]: Updated, and thanks for the reminder

**Pull request merged**